boms/cloud-oss-bom/RELEASING.md

```diff
@@ -18,11 +27,30 @@ Press Enter to open github.com in your browser...
 
 Do it. This grants the script permission to create a PR for you on Github.
 
-Ask a teammate to review the and approve the PR. While you should not push the final release until the PR is approved, you should kick off the  
+Ask a teammate to review the and approve the PR. 
+
+### Build the release binary with Rapid (CLI)
+
+While you should not push the final release until the PR is approved, you should kick off the  
 [Rapid build](https://rapid.corp.google.com/cloud-java-tools-cloud-opensource-java-bom-kokoro-release) while you wait for approval.
+
+Run gcert if you have not done so in the last twelve hours or so.
+
+```
+$ g4d -f bom-version
+$ blaze run java/com/google/cloud/java/tools:ReleaseBom -- --version=<release version>
+```
+
+### Build the release binary with Rapid (Legacy web UI)
```

Comment:
```suggestion
### Check the release status in Rapid web UI
```

I was thinking to suggest removing this section but we still need to check the web UI to see the progress.

Reply:
I'm not sure we need to check the UI. I need to figure out how the script can query the status to warn the user if something goes wrong. At that point it can provide a link to Rapid or the logs.

Reply:
> At that point it can provide a link to Rapid or the logs.

That would be nice.